### DISCUSSION TITLE
How to fix "the system cannot find the path specified"

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2545] [Q&A]
This error makes "lime test windows" work like "lime build windows" for me.
So i have to launch the game myself

And this makes trace functions not working, and they are needed for debugging.

ANY idea, please?

**Comment by NikRetaNCAM** [comment]
I'll leave a link for you to understand what happens
https://youtu.be/ddvszKkaTK0

**Reply by RichardBray** [reply]
Are you able to test and debug in HTML5?

I usually find that faster during the test phase of game development.
